sdk/resources/arm-resources/test/cae_mock_test.spec.ts

```diff
@@ -50,16 +54,16 @@ describe("Mock test for CAE with ResourceManagementClient", () => {
     assert.deepEqual(request!.headers.get("authorization"), "Bearer testToken");
   });
 
-  it("should not proceed CAE process for mgmt client if an invalid CAE challenge", async function () {
+  it("should not proceed CAE process for mgmt client if an invalid CAE challenge", async () => {
     let getTokenCount = 0;
     const credential: TokenCredential = {
-      getToken: async (scopes) => {
+      getToken: async () => {
         getTokenCount++;
         let token = "testToken";
         if (getTokenCount === 0) {
           token = "firstToken";
         }
-        return { token: "testToken", expiresOnTimestamp: 11111 };
+        return { token: token, expiresOnTimestamp: 11111 };
```

Comment:
Is this expected. @MaryGao Could you help take a look? Thanks

Reply:
yes, this is expected and also we should upate the line63 with following
```
if (getTokenCount === 1) {
```

Reply:
updated